starfish/imagestack/imagestack.py

```diff
@@ -26,6 +26,7 @@
 from starfish.errors import DataFormatWarning
 from starfish.experiment.builder import build_image, TileFetcher
 from starfish.experiment.builder.defaultproviders import OnesTile, tile_fetcher_factory
+from starfish.imagestack import physical_coordiante_calculator
```

Comment:
s/physical_coordiante_calculator/physical_coordinate_calculator/

Reply:
Does this mean move this to it's own directory?

Reply:
No, there is a typo. :)

Reply:
OHHH my bad haha